e2e/test/scenarios/embedding/embedding-full-app.cy.spec.js

```diff
@@ -315,6 +326,64 @@ describeEE("scenarios > embedding > full app", () => {
 
       cy.findByTestId("main-logo").should("be.visible");
     });
+
+    it("should have parameters header occupied the entire horizontal space when visiting a dashboard via navigation (metabase#30645)", () => {
+      const dashboardDetails = {
+        name: "interactive dashboard embedding",
+        parameters: [
+          {
+            id: "50c9eac6",
+            name: "ID",
+            slug: "id",
+            type: "id",
+          },
+        ],
+      };
+      cy.createDashboard(dashboardDetails).then(
+        ({ body: { id: dashboardId } }) => {
+          const card = createTextCard({
+            col: 0,
+            row: 0,
+            size_x: 6,
+            size_y: 20,
+            text: "I am a very long text card",
+          });
+          updateDashboardCards({ dashboard_id: dashboardId, cards: [card] });
+        },
+      );
+
+      visitFullAppEmbeddingUrl({ url: "/" });
+
+      cy.log("Navigate to a dashboard via in-app navigation");
+      navigationSidebar().findByText("Our analytics").click();
+      cy.findByRole("main").findByText(dashboardDetails.name).click();
+      closeNavigationSidebar();
+      navigationSidebar().findByText("Our analytics").should("not.be.visible");
+
+      cy.get("main header")
+        .findByText(dashboardDetails.name)
+        .should("be.visible");
+      getDashboardCard()
+        .findByText("I am a very long text card")
+        .should("be.visible");
+
+      // The bug won't appear if we scroll instantly and check the position of the dashboard parameter header.
+      // I suspect that happens because we used to calculate the dashboard parameter header position in JavaScript,
+      // which could take some time.
+      const FPS = 1000 / 60;
+      cy.findByRole("main").scrollTo("bottom", { duration: 2 * FPS });
+
+      getDashboardCard()
+        .findByText("I am a very long text card")
+        .should("not.be.visible");
+      cy.findByTestId("dashboard-parameters-widget-container").then(
+        $dashboardParameters => {
+          const dashboardParametersRect =
+            $dashboardParameters[0].getBoundingClientRect();
+          expect(dashboardParametersRect.x).to.equal(0);
```

Comment:
expects that the dashboard parameter's `x` should be `0` meaning that it's occupying the entire screen width.

#### Issue (you could see that there is a huge gap on the left side of the dashboard parameters)
![0 46 1 test](https://github.com/metabase/metabase/assets/1937582/3aacfe12-8012-4a50-8478-3aef80e98f4e)

#### Fix
![master@9c4abbb8dddba5f41c418df698d11258d9b08b05](https://github.com/metabase/metabase/assets/1937582/9478149d-50dd-4302-afa8-2b4c4347ea79)